src/main/java/uk/gov/hmcts/reform/bsp/common/model/document/Addressee.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.bsp.common.model.document;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Builder
+@EqualsAndHashCode
+public class Addressee {
+    @JsonProperty("name")
```

Comment:
I'd call it addressee name (unless the adressee class is an element in itself).

Reply:
yeah, the plan is to have: `addressee.name`